src/index.js

```diff
@@ -46,6 +47,10 @@ try {
 }
 catch (e) { }
 
+if (DEV && nativeCreateContext) {
```

Comment:
We support the `createContext` API here via `preact-context`. Maybe we could check the vnode shape instead. One way to do it, would be via the return value of `h("div", null).type === "div"` or something similar.